Sources/_SwiftSyntaxTestSupport/SyntaxComparison.swift

```diff
@@ -76,14 +76,17 @@ extension TreeDifference: CustomDebugStringConvertible {
                 """
     }
 
+    let baselineRootConverter = SourceLocationConverter(file: "Baseline.swift", source: baseline.root.description)
+    let nodeRootConverter = SourceLocationConverter(file: "Actual.swift", source: node.root.description)
```

Comment:
May as well use these when printing the nodes above as well?

Reply:
Good idea 👍